frontend/test/metabase/scenarios/onboarding/reference/databases.cy.spec.js

```diff
@@ -77,6 +77,25 @@ describe("scenarios > reference > databases", () => {
       checkQuestionSourceDatabasesOrder("Native query");
     });
   });
+
+  it("should show a popover with metadata related to the table", () => {
+    cy.visit("/browse/1");
+    cy.findByText("Products").trigger("mouseenter");
+
+    popover().within(() => {
+      // check that is has shows the table's description
```

Comment:
```suggestion
      // check that is has shown the table's description
```

?

Reply:
besides the grammar error, the sentence felt a bit verbose, so i chopped it down a bit